### DISCUSSION TITLE
Deck.gl can't create WebGL context

### DISCUSSION THREAD

**Discussion by andreas-groos** [checkly/.github#4] [Product previews & research]
In every test [Deck.gl](https://deck.gl/) fails to create a WebGl context. This is does not occur in the actual application in the browser.
It is also the main issue I want to check (loading time etc).
Is there a way for Checkly to enable this? Do I need additional configuration?

![Screen Shot 2022-09-29 at 9 15 40 AM](https://user-images.githubusercontent.com/19787368/193084302-984a39cd-54a6-4bca-8576-ee17fac45357.png)

**Comment by tnolet** [comment]
@andreas-groos sorry for just noticing this. Did you manage to solve this?

**Reply by andreas-groos** [reply]
Yes, with the help of some of your team. It's a known issue, and downgrading the runtime solves it. Thanks for checking.
